tracing/src/lib.rs

```diff
@@ -389,6 +388,11 @@ pub use self::{
     },
 };
 
+#[doc(inline)]
+pub use self::span::Span;
+#[doc(inline)]
+pub use tracing_attributes::instrument;
```

Comment:
EXTREMELY MINOR NIT

does this reexport it as `tracing::instrument` and if so are there any examples of using the `instrument` macro via this import? I'd personally find it slightly annoying if the only example of using the instrument macro had a path that was inconsistent with the path you'd use in the default recommended usecase.

Reply:
> does this reexport it as `tracing::instrument`
Yes

> and if so are there any examples of using the `instrument` macro via this import?

I was planning on adding examples in a follow-up branch that adds additional doc fixes. But, I'm fine with doing it here if you'd prefer.